js/app/interfaces/card.js

```diff
@@ -236,6 +239,24 @@ const Card = new Lang.Interface({
     },
 
     /**
+     * Method: add_style_classes_from_model
+     * Add style classes to the card depending on model properties
+     *
+     * This adds some CSS classes to cards based on model properties such as
+     * content type, that card implementations may want to use to customize the
+     * way they style themselves.
+     *
+     * Every card implementation's constructor should call this method after
+     * setting the construct properties.
+     */
+    add_style_classes_from_model: function () {
+        if (this.model.content_type === 'application/pdf') {
+            let clsname = Utils.get_modifier_style_class('Card', 'pdf');
```

Comment:
Ah you could also add another.
```Utils.get_modifier_style_class(this.constructor, 'pdf');```
This would give you a version of the modifier class specific to the card (its what Fernando did for the title and similar classes).

Then above in the themeing you could just `&--pdf` instead of `&.Card--pdf`